api/src/main/java/org/pmiops/workbench/actionaudit/ActionType.kt

```diff
@@ -11,5 +11,5 @@ enum class ActionType {
     DUPLICATE_TO,
     COLLABORATE,
     DELETE,
-    DETECT_HIGH_EGRESS_EVENT
+    DETECT_HIGH_EGRESS_EVENT,
```

Comment:
nit: I don't know if we have a decision on trailing commas in these kinds of lists. Seems like it might not work everywhere in Java.

Reply:
I've been adding these various places - especially `@Import({...})` lists in tests